### DISCUSSION TITLE
Is there any list of recommended textbooks for this course

### DISCUSSION THREAD

**Discussion by imflash217** [mlc-ai/mlc-en#51] [Q&A]
Hi,

Thank you so much amazing lectures. I was curious if there is any suggested list of textbooks to supplement this course. It will be greatly helpful. 

Thank you

**Comment by Edmon02** [comment]
Yes! Here are some recommended textbooks based on core machine learning concepts covered in this course:
	1.	“Pattern Recognition and Machine Learning” by Christopher M. Bishop – Great for statistical ML concepts.
	2.	“Deep Learning” by Ian Goodfellow, Yoshua Bengio, and Aaron Courville – Covers modern deep learning techniques.
	3.	“Hands-On Machine Learning with Scikit-Learn, Keras, and TensorFlow” by Aurélien Géron – A practical approach to ML.
	4.	“The Elements of Statistical Learning” by Hastie, Tibshirani, and Friedman – A deeper dive into the mathematical foundations of ML.

If you’re looking for specific topics, let me know, and I can suggest more books!
